Documentation/contributing/development/contributing_guide.rst

```diff
@@ -100,6 +100,17 @@ requirements have been met:
       the first 12 characters of the git SHA followed by the full commit title
       as seen above without breaking the line.
 
+#. If you change CLI arguments of any binaries in this repo, the CI will reject your PR if you don't
+also update the command reference docs. To do so, make sure to build the binaries, run the relevant script
+and commit the changes:
```

Comment:
Please keep the style from the other list items.
```suggestion
#. If you change CLI arguments of any binaries in this repo, the CI will reject your PR if you don't
   also update the command reference docs. To do so, make sure to build the binaries, run the relevant script
   and commit the changes:
```

Reply:
(Also I think the `.. code-block` and snippet should be indented? Would you mind double-checking please?)

Reply:
Doesn't work with indentation :(

Reply:
Hmm this is strange, we have indented blocks elsewhere in the doc (Documentation/gettingstarted/local-redirect-policy.rst or Documentation/policy/troubleshooting.rst for instance). Can you please check if the block is part if the item in the list, if not indented?

Reply:
Argh, I did not indent the content of the block :facepalm: . That did the trick!